### ISSUE TITLE
Examples Layout Docs

### ISSUE THREAD

**Issue by juanitorduz** [py-econometrics/pyfixest#541]
I could imagine many new examples coming up :) . The current layout makes this hard as they will fill the header right? 

![image](https://github.com/py-econometrics/pyfixest/assets/22996444/7d0eedb5-91c8-42c4-b92b-63097e80b098)

Is there any Quarto layout to let us have a "gallery view like" https://scikit-learn.org/stable/auto_examples/index.html ?

**Comment by s3alfisc** [comment]
Yes, I fully agree that it doesn't look amazing and that there are many more example notebooks to be written (e.g. I'd like to have a side by side comparison of r-fixest and py-fixest as @Wenzhi-Ding had suggested at one point). 

Adding a side bar should definitely be possible - the [quartodoc documentation](https://machow.github.io/quartodoc/get-started/overview.html), which is of course written in quartodoc (😄) has a nice sidebar panel. There also is an [example section ](https://machow.github.io/quartodoc/examples/) with more example pages. Let me know in case you spot one that you particularly like! =)

**Comment by juanitorduz** [comment]
> Btw, with which tool are the `scikit-learn` docs build? Is it `sphinx`?

I think so... I will try to see if we can get something similar in Quarto

**Comment by s3alfisc** [comment]
Closing in favor of #1174
